src/content/api/compiler-hooks.md

```diff
@@ -41,7 +41,7 @@ compiler.hooks.someHook.tap(/* ... */);
 
 Depending on the hook type, `tapAsync` and `tapPromise` may also be available.
 
-For the description of hook types, see [the Tapable docs](https://github.com/webpack/tapable#hook-types).
+For the description of hook types, see [the Tapable docs](https://github.com/webpack/tapable#tapable).
```

Comment:
This is a big change? What is the reason to drop the `-` for `.`?

Reply:
`https://github.com/webpack/tapable#hook-types` simply doesn't exist any more. Found the next best fragment to link to.

The removal  of `-` where there were dots in the original content is simply because that's the ouput of the markdown parser. I guess you switched something, so none of those fragment links were working any more